docs/reference/api/docker_remote_api_v1.22.md

```diff
@@ -922,6 +921,10 @@ Start the container `id`
 
     HTTP/1.1 204 No Content
 
+Query Parameters:
+
+-   **detacheys** – Specify the escape key sequence used to detach a container.
```

Comment:
`Specify a custom key sequence for detaching a container. Format is ctrl-<value> where <value> is one of: a-z, @, ^, [, ``, or _`